release-notes/8.0/8.0.0/cve.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This is a bit bare. I think we should include a date and .NET version.

Reply:
Yep, only the existing data so far. It's one of the points that I wanted to discuss what data could make sense in here.

Reply:
Affected (supported) versions would be good.

Reply:
We need a date in the front matter. If you follow the links from cve-index.json, there are no dates. You'd have to also read releases-index.json to get the dates. I think that's the only addition we need for CVEs.

I don't think we need "also applies to" unless we get asked for that. That is calculatable by looking at all the cve.json files for that date for supported versions.